roles/openshift_node/tasks/main.yml

```diff
@@ -44,6 +44,15 @@
 - name: include node installer
   include_tasks: install.yml
 
+- name: Restart cri-o
```

Comment:
Why does this need to be restarted here?  There aren't any changes being made to crio in this role as far as I can tell.

Reply:
This was required when the sdn was installed just prior to this section. Since ``sdn-ovs`` installation was moved to ``install.yml`` the restart is likely still required.

Ref: https://github.com/openshift/openshift-ansible/pull/6362#pullrequestreview-81978707